consul/src/main/java/com/linecorp/armeria/internal/consul/ConsulDefaults.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.internal.consul;
+
+/**
+ * Consul related constant values.
+ */
+public final class ConsulDefaults {
+    public static final int DEFAULT_CONNECT_TIMEOUT_MS = 1_000;
+    public static final int DEFAULT_SESSION_TIMEOUT_MS = 10_000;
+    public static final String DEFAULT_CHECK_INTERVAL = "10s";
```

Comment:
Now, I couldn't find any usage of this configuration. Let's remove this class.😀

Reply:
Ok! 😭